src/DynamoCoreWpf/Utilities/LibraryDragAndDrop.cs

```diff
@@ -29,18 +29,24 @@ internal void HandleMouseDown(Point position, NodeSearchElementViewModel node)
 
         internal void HandleMouseMove(FrameworkElement sender, Point currentPosition)
         {
-            if (isDragging || nodeViewModel == null)
+            if (isDragging)
```

Comment:
I moved this check to OnExpanderButtonPreviewMouseMove event.  Dragging will happen only when this vm is not null. 
Is the check redundant? Because, the nodeviewmodel gets value only when it is not null. (check in OnExpanderButtonMouseLeftButtonDown). I thought OnExpanderButtonPreviewMouseMove should have similiar check.